index.html

```diff
@@ -58,12 +61,347 @@ <h1>
     
             </div>
 
-        </section>
+        </section>     
+    </section>
+    <section class="recent-works">
+      <h1>My Recent Works</h1>
+      <div class="line-rule"></div>
+
+      <div class="img-placeholder">
+        <img src="assets/img-placeholder.png" alt=""/>
+      </div>
+
+      <div class="post">
+        <h2>Multi-Post Stories</h2>
+      </div>
+      <div class="content">
+        <h4>
+          A daily selection of privately personalized reads; 
+          no accounts or sign-ups required. 
+          has been the industry's standard dummy text ever since the 1500s, 
+          when an unknown printer took a standard dummy text.
+        </h4>
+      </div>
+      <div class="tag">
+        <h6>Css</h6>
+        <h6>Html</h6>
+        <h6>bootstrap</h6>
+        <h6>JavaScript</h6>
+      </div>
+      <div class="project-btn">
+        <h4>See Project</h4>
+      </div>      
```

Comment:
- [ ] Kindly center the text of your `see project` button to match the original design.

![image](https://user-images.githubusercontent.com/98453979/205310403-82249a9f-b77c-4f06-91eb-abb80003dfc5.png)
 
### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/205310544-59df8b27-b8ee-4328-bbbb-1d28963c7e9a.png)